pulsar-broker/src/test/java/org/apache/pulsar/broker/stats/ManagedCursorMetricsTest.java

```diff
@@ -22,10 +22,17 @@
 import org.apache.bookkeeper.client.PulsarMockLedgerHandle;
 import org.apache.pulsar.broker.auth.MockedPulsarServiceBaseTest;
 import org.apache.pulsar.broker.stats.metrics.ManagedCursorMetrics;
-import org.apache.pulsar.client.api.Consumer;
-import org.apache.pulsar.client.api.Producer;
+
```

Comment:
The code style used in Pulsar code base doesn't have empty line between imports. you might want to configure IntelliJ to use the Pulsar code style and checkstyle configuration. the instructions are in the README, https://github.com/apache/pulsar#configure-code-style

Reply:
done